### ISSUE TITLE
catmaid_get_connectors_between(post_skids = skid) returns mix of genuine and false connections

### ISSUE THREAD

**Issue by K-Meech** [jefferis/rcatmaid#76]
If enter post_skids = some skid and leave pre_skids empty seems to return a mix of genuine and false connections. e.g. for skid 2477473 the second returned is 480347 which is not connected.

**Comment by jefferis** [comment]
To add some details for this:

```r
> posts=catmaid_get_connectors_between(post_skids = 2477473)
> head(posts)
  pre_skid post_skid connector_id pre_node_id post_node_id connector_x connector_y connector_z
1    61221   2477473      2556847     2556845      2556851      436134      236163      135765
2   480347   2477473      2556847     2556849      2556851      436134      236163      135765
3   480352   2477473      2556847     2556850      2556851      436134      236163      135765
4   480362   2477473      2556847     2556852      2556851      436134      236163      135765
5   480367   2477473      2556847     2556854      2556851      436134      236163      135765
6  1853423   2477473      2556847     6665855      2556851      436134      236163      135765
  pre_node_x pre_node_y pre_node_z post_node_x post_node_y post_node_z pre_confidence pre_user
1     436270     236445     135765      436057      235975      135765              5       75
2     436337     235928     135765      436057      235975      135765              5       75
3     436184     235976     135765      436057      235975      135765              5       75
4     436047     236056     135765      436057      235975      135765              5       75
5     435934     236211     135765      436057      235975      135765              5       75
6     435817     236306     135695      436057      235975      135765              5      117
  post_confidence post_user
1               5        75
2               5        75
3               5        75
4               5        75
5               5        75
6               5        75

> ct=catmaid_get_connector_table(2477473)
> head(ct)
  connector_id    skid      x      y      z confidence user_id partner_treenode_id
1      2556847 2477473 436134 236163 135765          5      75             2556851
2      2570784 2477473 420902 239109 135590          5      75             2570785
3      2570837 2477473 420503 240325 135975          5      75             2570839
4      2570864 2477473 420459 240017 135695          5      75             2570869
5      2590073 2477473 420206 240274 135275          5      75             2590078
6      2625841 2477473 436355 236090 135310          5      75             2625842
                     last_modified partner_skid direction
1 2017-01-20T12:36:04.682268+00:00        61221  incoming
2 2017-01-20T12:36:04.682268+00:00        61221  incoming
3 2017-02-06T14:30:23.605955+00:00        61221  incoming
4 2017-02-06T14:31:11.594297+00:00        61221  incoming
5 2017-02-06T14:25:49.053129+00:00        61221  incoming
6 2017-01-20T12:36:04.682268+00:00       755022  incoming
> subset(ct, connector_id==2556847)
  connector_id    skid      x      y      z confidence user_id partner_treenode_id
1      2556847 2477473 436134 236163 135765          5      75             2556851
                     last_modified partner_skid direction
1 2017-01-20T12:36:04.682268+00:00        61221  incoming
```

**Comment by jefferis** [comment]
Thanks @K-Meech. See https://github.com/catmaid/CATMAID/issues/1554 for the gory details. I'm not sure if I'm misusing the API endpoint that they've provided in a way that they did not intend or if they would agree that this a CATMAID bug. Seems like one to me. Best, Greg.

**Comment by jefferis** [comment]
@tomka agreed this was an API bug and fixed it – see catmaid/CATMAID#1554. It will be fixed for us the next time the update the FAFB catmaid server. Well spotted @K-Meech!

**Comment by tomka** [comment]
Yep, thanks very much @K-Meech for reporting this!
